sklearn/decomposition/nmf.py

```diff
@@ -434,7 +434,7 @@ def _fit_projected_gradient(X, W, H, tol, max_iter,
                                                        alpha, l1_ratio,
                                                        sparseness, beta, eta)
         if iterW == 1:
-            tolW = 0.1 * tolW
```

Comment:
changes like these are not great, as they create conflicts in pull requests and don't provide any benefit